package.json

```diff
@@ -9,7 +9,7 @@
     "@types/react": "^17.0.39",
     "cypress": "^9.5.0",
     "eslint-plugin-cypress": "^2.12.1",
-    "prisma": "^4.2.1",
+    "prisma": "^4.13.0",
```

Comment:
Please don't modify this file, a different version can affect other dependencies, so we don't want to take that risk.